weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -972,8 +1007,9 @@ def _select_objs_query(
                 version_count,
                 is_latest
             FROM object_versions_deduped
-            WHERE project_id = {{project_id: String}}
-            AND {conditions_part}
+            WHERE project_id = {{project_id: String}} AND
+                deleted_at IS NULL AND
```

Comment:
just adding this in anticipation?? (:

Reply:
sorry yeah, i have another pr that this should be moved to ;)